### PR TITLE
Change wercker deployment to GKE 

### PR DESCRIPTION
**Pull request by scheeles**

### CODE REVIEW

**Comment by scheeles** [comment]
@sttts this is now the correct one

File: wercker.yml [hunk 1]
```diff
@@ -57,13 +59,17 @@ deploy:
...
+        insecure-skip-tls-verify: true
         command: rolling-update --namespace=api --poll-interval=500ms --update-period=2s kubermatic-api-v1
         image: kubermatic/api:$WERCKER_GIT_COMMIT
     - kubectl:
-        server: https://do-ams2.kubermatic.io
-        token: $KUBERNETES_TOKEN_DO_AMS2
+        server: $KUBERNETES_MASTER
+        username: $KUBERNETES_USERNAME
+        password: $KUBERNETES_PASSWORD
+        insecure-skip-tls-verify: true
```
Comment (sttts) [inline wercker.yml]: Why insecure-skip-tls-verify ?

Comment (scheeles) [inline wercker.yml]: How can I use certificates with wercker. The certificates are stored in kubeconfig and all examples I found use insecure-skip-tls-verify .
The GKE cluster endpoint is only an IP and don't have a valid URL.

Comment (sttts) [inline wercker.yml]: Oh, good question. I understand now. Can't one place the root ca in the repo and access it from the deploy container?

Comment (scheeles) [inline wercker.yml]: I will investigate what is possible. I will contact the wercker guys if they have a solution. We have today our Amsterdam meetup there.

Comment (scheeles) [inline wercker.yml]: Ohh seems to work with wercker 2.1.0 https://github.com/wercker/step-kubectl/blob/master/README.md 
I will update it

Comment (scheeles) [inline wercker.yml]: It is fixed now. I know i have both client cert and user/password. It is more for documentation, so we know which parameter can be set instead as both are required.

**Pull request merged**